server/api/crud/projects.py

```diff
@@ -475,14 +488,7 @@ async def _calculate_pipelines_counters(
             # project_to_recent_failed_pipelines_count, project_to_running_pipelines_count,
             # in case of exception we want to return 3 * defaultdict of None because this function
             # returns 3 values
-            return [collections.defaultdict(lambda: None)] * 3
-
-        for pipeline in pipelines:
-            if (
-                pipeline["status"]
-                not in mlrun_pipelines.common.models.RunStatuses.stable_statuses()
-            ):
-                project_to_running_pipelines_count[pipeline["project"]] += 1
+            return [collections.defaultdict(lambda: 0)] * 3
```

Comment:
why?

Reply:
Why did I removed it?
Because we already count it here:
https://github.com/mlrun/mlrun/pull/5535/files#diff-cd538675149b5763b087a3f87325f4cc716889ac78b2312f793430f8129a3b0aL442-R459